devicetypes/smartthings/smartsense-motion-sensor.src/smartsense-motion-sensor.groovy

```diff
@@ -230,30 +230,46 @@ def getTemperature(value) {
 }
 
 private Map getBatteryResult(rawValue) {
-	log.debug 'Battery'
+	log.debug "Battery rawValue = ${rawValue}"
 	def linkText = getLinkText(device)
 
-	log.debug rawValue
-
 	def result = [
 		name: 'battery',
 		value: '--'
 	]
 
 	def volts = rawValue / 10
-	def descriptionText
 
-	if (rawValue == 0) {}
+	if (rawValue == 255) {}
```

Comment:
Technically 0 is a valid value, but if it is actually 0 then it's wrong because the sensor couldn't possibly be powered. I think I'm going to update all the sensors to report '--' if rawValue is 0 or 255